### PR TITLE
Vtuhandler

### PR DESCRIPTION
**Pull request by twisterbboy**
I think that for this type of file `VTU` whose internal mesh we want to display we need a `tetradata` instead of a `polydata`. So `VTK_TETRA` should be imported instead of `VTK_POLYGON` and used in the `write()` method as `unstructured_grid.SetCells(VTK_TETRA, cells)`. Moreover, only by removing the geometric filter is it possible to correctly display the internal mesh on the paraview by sectioning it, for example, with a slice.

### CODE REVIEW

File: pydata/vtuhandler.py [hunk 1]
```diff
@@ -27,22 +26,18 @@ class VTUHandler(VTKHandler):
     @classmethod
     def _polydata_from_file(cls, filename):
         """
-        Private method to extract vtkPolyData from `filename`. The `filename`
+        Private method to extract vtkTetraData from `filename`. The `filename`
```
Comment (ndem0) [inline pydata/vtuhandler.py:29]: Seriously: what the hell is `vtkTetradata`? No a single match in all the VTK documentation.

File: pydata/vtuhandler.py [hunk 2]
```diff
@@ -27,22 +26,18 @@ class VTUHandler(VTKHandler):
...
         have to be well-formatted VTU file.
 
         :param str filename: the name of the file to parse.
         :return: the dataset
-        :rtype: vtkPolyData
+        :rtype: vtkTetraData
```
Comment (ndem0) [inline pydata/vtuhandler.py:34]: Same as before.

File: pydata/vtuhandler.py [hunk 3]
```diff
@@ -27,22 +26,18 @@ class VTUHandler(VTKHandler):
...
         reader = cls._reader_()
         reader.SetFileName(filename)
         reader.Update()
 
-        geometryFilter = vtkGeometryFilter()
-        geometryFilter.SetInputData(reader.GetOutput())
-        geometryFilter.Update()
-
-        return geometryFilter.GetOutput()
+        return reader.GetOutput()
```
Comment (ndem0) [inline pydata/vtuhandler.py:40]: I really don't understand how, removing 4 lines, we obtain the same results (points and faces) as before.

File: pydata/vtuhandler.py (hunk 1 above)
Comment (twisterbboy) [reply pydata/vtuhandler.py:29]: Yes. you're absolutely right. there is no such library. I remove it on the next push.

File: pydata/vtuhandler.py (hunk 3 above)
Comment (twisterbboy) [reply pydata/vtuhandler.py:40]: I don't know this geometric filter, but leaving `reader.GetOutput()` I get everything I need. So I thought about removing the filter.

Comment (ndem0) [reply pydata/vtuhandler.py:40]: Again: test your implementation by converting a VTK to a VTU file (and viceversa). I guess you did not get what you need.
PS. if you are developing a piece of code with VTK library, you HAVE TO KNOW what are the variables you are removing.

**Pull request closed without merging**